plugins/inputs/ethtool/ethtool_test.go

```diff
@@ -305,6 +305,15 @@ func toStringMapInterface(in map[string]uint64) map[string]interface{} {
 	return m
 }
 
+func toStringMapUint(in map[string]interface{}) map[string]uint64 {
+	var m = map[string]uint64{}
+	for k, v := range in {
+		t := v.(uint64)
+		m[k] = t
```

Comment:
This will panic if something is not an uint.... Are we _really_ sure this doesn't fire back at us?

Reply:
I added an error. It is only used in the test case `TestNormalizedKeys` since it is no longer a unit64 as it needs to contain a boolean. But `InterfaceMock` requires `stats` to be unit. If you have other ideas to avoid this I am open to hearing them :)